test/config/karma.conf.js

```diff
@@ -6,6 +6,11 @@ module.exports = function(config) {
   const singleRun = process.env.SINGLE_RUN !== 'false';
 
   config.set({
+    client: {
+      mocha: {
+        timeout: 5000
```

Comment:
why do we need this? A comment in code will be great if in the future we need to remember why we did this

Reply:
Added a explanatory comment.